SweetPea to English/gpt3prompt_regular_factors.txt

```diff
@@ -60,3 +60,66 @@ Text:
 There are four regular factors: color, sounds, letters, and numbers. The color factor consists of three levels: "blue", "green", "red". The sounds factor consists of two levels: "bell" and "honk". The letters factor consists of three levels: "a", "b", "c". The numbers factor consists of five levels: "1", "2", "3", "4", and "5".
 
 Code:
+object = Factor("object identity", [Level("H", 2), Level("S", 1)])
+motion = Factor("motion", ["left", "right"])
+
+Text:
+There are two regular factors: object identity and motion. The object identity factor consists of two levels: "H" and "S". In the object identity factor, the ratio between the levels "H" and "S" is 2 to 1. The motion factor also consists of the two levels: "left" and "right".
+
+Code:
```

Comment:
Generally, we might want to change the factor & level names for each prompt, so GPT-3 doesn't get overtrained on Stroop...